filebeat/input/syslog/rfc3164_test.go

```diff
@@ -793,7 +793,6 @@ func TestPriority(t *testing.T) {
 			ParserRFC3164([]byte(log), l)
 			assert.Equal(t, d, l.Priority())
 		})
-		return
```

Comment:
This looks suspicious and it changes how the test run. Before it was running only once and now for 120 iterations. I think it's safer to remove the loop in order to preserve the existing logic.

Reply:
> This looks suspicious and it changes how the test run. Before it was running only once and now for 120 iterations. 

Yes. IMO, that was the intent of this test, only the `return` statement was added by mistake

> I think it's safer to remove the loop in order to preserve the existing logic.

I'm keeping an open mind on this.
I'll wait a few hours, and if no other revier has a problem with it, I'll delete the loop.
@efd6 @ShourieG @cmacknz

Reply:
Running the loop only once is not correct IMO. The point of the test is to check that a variety of syslog priorities are properly parsed into the destination value. Testing only a single value (1) is not interesting. I'm not entirely sure why it starts at 1 and stops at 120 since it would not cost significantly more to test the exhaustive range of valid priority values.